src/loki/reach.cc

```diff
@@ -13,6 +13,28 @@ Reach::Reach() : Dijkstras() {
   path_edge->set_end_node(false);
 }
 
+void Reach::enqueue(const baldr::GraphId& node_id,
+                    baldr::GraphReader& reader,
+                    const std::shared_ptr<sif::DynamicCost>& costing,
+                    const baldr::GraphTile* tile) {
```

Comment:
i moved this method up for visibility since its called right below here. the only change was to not pass the pointer by reference so that when the end node that we are enqueing is in another tile, we wont modify it. modifying it screws over the costing->filter call that happens inside the node expansion loop in the operator() below